cluster/src/main/java/org/apache/iotdb/cluster/log/logtypes/AddNodeLog.java

```diff
@@ -82,11 +120,17 @@ public boolean equals(Object o) {
       return false;
     }
     AddNodeLog that = (AddNodeLog) o;
-    return Objects.equals(newNode, that.newNode);
+    return Objects.equals(newNode, that.newNode)
```

Comment:
Because sub-class has its own variables, it's necessary to rewrite these two functions for completeness. All sub-class of class Log have rewritten these two functions.